lib/src/widgets/switch/ds_switch_tile.widget.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DsSwitchTile extends StatelessWidget {
+  /// Create a tile widget with a switch button
+  ///
+  ///It is mandatory to inform three parameters: [value], enabling or disabling the switch, [onChanged], the event that will be executed when changing the value, and [title], which is the title of the tile.
+  const DsSwitchTile({
+    Key? key,
+    required this.value,
+    required this.onChanged,
+    required this.title,
+    this.switchScale = 1.0,
+    this.switchActiveColor = DSColors.primaryNight,
+    this.switchInactiveColor = DSColors.neutralMediumSilver,
+    this.leading,
+    this.isEnabled = true,
+    this.contentPadding,
+  })  : assert(switchScale <= 1.5),
+        super(key: key);
+
+  /// current state of the switch widget.
+  final bool value;
+
+  /// Similar to [Switch.onChanged] and [CupertinoSwitch.onChanged].
+  final ValueChanged<bool> onChanged;
+
+  final Widget? title;
+
+  /// Indicates the size of the switch button that will be displayed.
+  final double switchScale;
+
+  /// Switch button and title line color when [value] is true.
+  final Color? switchActiveColor;
+
+  /// Switch button color when [value] is false.
+  final Color? switchInactiveColor;
+
+  /// Make it possible to use an image, icon, avatar or other widget together with the switch before the title.
+  final Widget? leading;
+
+  /// Enables or disables the widget by changing the color of the switch.
+  final bool isEnabled;
+
+  /// Change a padding tile content.
+  final EdgeInsetsGeometry? contentPadding;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTileTheme.merge(
+      selectedColor: switchActiveColor,
+      child: ListTile(
+        enabled: isEnabled,
+        onTap: () => onChanged(!value),
+        leading: leading,
+        title: title,
+        trailing: Builder(
+          builder: (_) => Transform.scale(
+            alignment: Alignment.center,
+            scale: switchScale,
+            child: CupertinoSwitch(
```

Comment:
We'll have to create a custom switch. CupertinoSwitch does not fit the prototype.

Switch from Figma:
<img width="148" alt="Screen Shot 2022-08-09 at 17 20 26" src="https://user-images.githubusercontent.com/20003773/183753142-62e9355e-f6ac-4059-834a-92ebef4009cd.png">

CupertinoSwitch:
<img width="71" alt="Screen Shot 2022-08-09 at 17 02 15" src="https://user-images.githubusercontent.com/20003773/183753214-659a219a-0218-4dab-a34f-489d5381af22.png">

Reply:
In the refinement we present the cupertino switch. At the time this was not an impediment. If so, the code will be redone, creating a custom switch.

Reply:
In conversation with Torugo, he signaled that the layout was different. I believe that the ideal is to create a switch so that the layout is in accordance with the prototype. We don't pay attention to this in refinement.

Reply:
Done.